features/nanostack/targets/TARGET_NXP/TARGET_KW41Z/NanostackRfPhyKw41z.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2018 ARM Limited. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ * Licensed under the Apache License, Version 2.0 (the License); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef NANOSTACK_PHY_KW41Z_H_
+#define NANOSTACK_PHY_KW41Z_H_
+
+#include "mbed.h"
```

Comment:
please include what is needed, not `mbed.h`

Reply:
I got this from other supported targets, 
https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/targets/TARGET_NCS36510/NanostackRfPhyNcs36510.h#L20
https://github.com/ARMmbed/mbed-os/blob/master/features/nanostack/targets/TARGET_Silicon_Labs/TARGET_SL_RAIL/NanostackRfPhyEfr32.h#L20